LDMP/gui/WidgetMain.ui

```diff
@@ -138,7 +156,7 @@
   <customwidget>
    <class>QgsFilterLineEdit</class>
    <extends>QLineEdit</extends>
-   <header>qgis.gui</header>
+   <header>qgsfilterlineedit.h</header>
```

Comment:
is it necessary this change?

Reply:
Qt Designer made this change, not sure what it means.

Reply:
going to revert this @luipir